src/pgo_math.erl

```diff
@@ -0,0 +1,108 @@
+pow10(72) -> 1000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(73) -> 10000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(74) -> 100000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(75) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(76) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(77) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(78) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(79) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(80) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(81) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(82) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(83) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(84) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(85) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(86) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(87) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(88) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(89) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(90) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(91) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(92) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(93) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(94) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(95) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(96) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(97) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(98) -> 100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(99) -> 1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
+pow10(100) -> 10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000;
```

Comment:
I feel a little dirty with this, hehe. Is it really worth it? But I suppose since it is already written may as well keep it :)

Reply:
Well, the purpose is to avoid floats and I suppose the C behind math:pow/2 while keeping things quick for at least 0-100 scales. It doesn't have to stay though. Up to you, but as example the code was having to do a `ceil(trunc(float * math:pow(...` now it does not... whether it's all worth it or not *shrug*.